Makefile

```diff
@@ -19,7 +27,12 @@ coverprofile_html:
 	go tool cover -html=coverage.out
 
 lint:
-	golangci-lint run --fix
+	golangci-lint run --fix --timeout 10m
+
+fmt:
+	prettier --check '**/**.{ts,js,md,yaml,yml,sass,css,scss,html}'
+	golines --max-len=88 --base-formatter=gofumpt -w $(GO_SOURCES)
```

Comment:
Why 88?

Reply:
It is based on [python black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length) (and _[Raymond Hettinger](https://youtu.be/wf-BqAjZb8M?t=260)_), which some might hate.

The idea is to make the code more naturally fit on narrow screens, so you can have multiple files open without bad editor wrapping.

Reply:
I'll adda note in the reader about code style

Reply:
Done